packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -492,15 +508,25 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
     payload?: ClientEventPayload;
     options?: SubmitClientEventOptions;
   }) {
-    const {meetingId, correlationId} = options;
+    const {meetingId, correlationId, rawError} = options;
     let clientEventObject: ClientEvent['payload'];
 
+    // check if we need to generate errors
+    const errors: ClientEvent['payload']['errors'] = [];
```

Comment:
Could we create a type ClientEventErrors in `metrics.types.ts` and use that everywhere

Reply:
if added ClientEventPayloadError bc ClientEventErrors already existed as a type